allensdk/brain_observatory/ecephys/stimulus_table/naming_utilities.py

```diff
@@ -195,4 +195,24 @@ def map_column_names(table, name_map=None, ignore_case=True):
     output = table.rename(columns=name_map)
 
     return output
-#
+
+
+def eval_str(val):
+    """Evaluates value if its type==str(numeric) or type==str(list)
+
+    Parameters
+    ----------
+    val: any
+
+    Returns
+    -------
+    val: evaluated val if its type==str(numeric) or type==str(list)
+    or passes the val through if it does not meet the condition
+    """
+
+    if isinstance(val, str):
+        if val.replace('.', '').isdigit():  # checks if val is numeric
```

Comment:
There can be issues if a string of characters are inputted into eval, like with the column `stimulus_name`. 
The alternative is to specify which columns you want to eval. I thought coming up with a set of rules to decide which column to eval instead of explicitly naming a list of them would make it easier to maintain.